classes/class-admin.php

```diff
@@ -469,14 +484,10 @@ public function admin_enqueue_scripts( $hook ) {
 				'wp_stream_live_updates',
 				array(
 					'current_screen'      => $hook,
-					'current_page'        => isset( $_GET['paged'] ) ? esc_js( $_GET['paged'] ) : '1', // WPCS: CSRF ok.
-					// input var okay, CSRF okay
-					'current_order'       => isset( $_GET['order'] ) ? esc_js( $_GET['order'] ) : 'desc', // WPCS: CSRF ok.
-					// input var okay, CSRF okay
-					'current_query'       => wp_stream_json_encode( $_GET ), // WPCS: CSRF ok.
-					// input var okay, CSRF okay
-					'current_query_count' => count( $_GET ), // WPCS: CSRF ok.
-					// input var okay, CSRF okay
+					'current_page'        => isset( $_GET['paged'] ) ? esc_js( $_GET['paged'] ) : '1', // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
Should this be sanitized to an integer value?

Reply:
`absint( wp_unslash( $_GET['paged] ) )` would probably be my preference here, but there are many ways to sanitize/escape this.

Reply:
I gotcha, I wasn't worried about the code and was just trying to silence the PHPCS warning.